src/components/screens/layouts-screen.tsx

```diff
@@ -148,21 +147,20 @@ class LayoutsScreen extends React.Component<Props, State> {
       return (
         <CardItem
           key={ subLayout.id }
-          title={ `${strings.subLayout.title} - ${subLayout.name}` }
+          title={ `${ strings.subLayout.name } - ${ subLayout.name }` }
```

Comment:
String literals are written without spaces around the variables within string
```suggestion
          title={ `${strings.subLayout.name} - ${subLayout.name}` }
```